packages/ethereum-storage/src/ethereum-storage.ts

```diff
@@ -225,6 +225,49 @@ export default class EthereumStorage implements StorageTypes.IStorage {
     };
   }
 
+  /**
+   * Append content into the storage: add the content to ipfs and the hash on Ethereum
+   * @param content Content to add into the storage
+   * @returns Promise resolving id used to retrieve the content
+   */
+  public async _ipfsAdd(data: string): Promise<StorageTypes.IIpfsMeta> {
```

Comment:
just curious, don't we have this anywhere else? adding stuff to IPFS isn't new...

Reply:
Normally, ipfs is called internally in `ethereum-storage` the newness is the fact that it is in the interface now.

Reply:
OK but this function is not called by the existing code, is it?